dev/build_release.py

```diff
@@ -236,14 +236,24 @@ def start_container_build(self, name):
                 project=self.__options.gcb_project)
       )
     elif self.__options.container_builder == 'docker':
-      return BackgroundProcess.spawn(
-          'Build/publishing container image for {name} with Docker...'.format(
+      if name == 'spinnaker-monitoring':
```

Comment:
also, how does third-party (for dashbaords and integrations) factor into docker?